ci/docker/init.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+uid=$(id -u)
+gid=$(id -g)
+
+printf "UID=${uid}\nGID=${gid}\nCOMPOSE_PROJECT_NAME=eb" > .env
```

Comment:
COMPOSE_PROJECT_NAME is set to: `eb`. That does not make sense right? Is the init.sh script still used?